randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -233,16 +233,40 @@ Extra - room_id: [9]
   > Pickup (Missile Tank)
       All of the following:
           Morph Ball
-          Can Activate Pillar or Can climb High Jump Wall
+          Any of the following:
+              Can Activate Pillar or Can climb High Jump Wall
+              Wall Jump (Beginner) and Can Freeze Enemies
   > Door to Maintenance Annex
-      Can Use Any Bombs
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Morph Ball Bombs
+              All of the following:
+                  Knowledge (Advanced) and Normal Damage ≥ 88
+                  Any of the following:
+                      No Plasma Beam
+                      Diffusion Missiles and No Ice Missiles and No Missile Launcher and No Super Missiles
```

Comment:
There will be an option to turn off items so there is no need to have `no item` requirement.

The generator HATES these statements. Please remove and avoid these in the future.

Reply:
removed